### PR TITLE
document Unity XR testing (fixes issue #155)

### PR DESCRIPTION
**Pull request by cvan**
(PNGs are compressed)

### CODE REVIEW

File: docs/xr-testing.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Testing your Unity project in Virtual Reality
+
+## XR Settings
+
+Enabling Virtual Reality support for testing in VR mode your [Unity project](./project-setup.md):
+
+![Animation of Unity's XR Settings](https://raw.githubusercontent.com/cvan/unity-webvr-export/docs-xr-testing/docs/images/unity-xr-settings.gif "Animation of Unity's XR Settings")
+
+1. Select the **`File > Build Settings…`** menu option from the top-level Unity application menu.
+2. When the pop-up window appears, select the **`PC, Mac & Linux Standalone`** option in the **`Platform`** list, and click the **`Switch Platform`** button.
```
Comment (cvan) [inline docs/xr-testing.md]: I just realised you don't have to actually click the **`Switch Platform`** button. you have to only highlight the **`PC, Mac & Linux Standalone`** option in order for the **`Inspector > XR Settings`** pane to show up. :+1:

will remove the screenshot, regenerate the GIF, update this instruction (2), and remove the last instruction (8). that simplifies things a bit. sweet! 👍

**Comment by cvan** [comment]
@caseyyee this is updated - can you take another review? you can view the rendered [Markdown doc here](https://github.com/cvan/unity-webvr-export/blob/docs-xr-testing/docs/xr-testing.md).

File: docs/xr-testing.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+5. Click on the **`XR Settings`** title to expose the section if it is hidden. Enable the **`Virtual Reality Supported`** checkbox field.
+
+    ![Screenshot of Unity's "XR Settings" ](./images/unity-xr-settings-7.png 'Screenshot of Unity\'s "XR Settings"')
+
+6. Notice that are two **`Virtual Reality SDKs`** listed:
+    - **`OpenVR`** (i.e., for the HTC Vive and Windows Mixed Reality headsets)
+    - **`Oculus`** (i.e., for the Oculus Rift and, soon, Oculus Go headsets)
+
+    The VR devices are sorted in the order in which Unity tries to enable VR devices at runtime or during VR preview.
```
Comment (delapuente) [inline docs/xr-testing.md]: Could you add a reminder about "double checking the platform is still WebGL" ?

File: docs/xr-testing.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Testing your Unity project in Virtual Reality
```
Comment (caseyyee) [inline docs/xr-testing.md]: Maybe being a bit more specific will be good here?  Something like:
_"Enable Unity XR support for faster testing within Unity editor"_

**Pull request merged**